src/vs/workbench/contrib/notebook/common/notebookCommon.ts

```diff
@@ -1018,7 +1018,7 @@ export const NotebookSetting = {
 	scrollToRevealCell: 'notebook.scrolling.revealNextCellOnExecute',
 	cellChat: 'notebook.experimental.cellChat',
 	cellGenerate: 'notebook.experimental.generate',
-	notebookVariablesView: 'notebook.experimental.variablesView',
+	notebookVariablesView: 'notebook.variablesView',
```

Comment:
@amunger since we are renaming it, I wonder if we could make it future proof `notebook.variablesView.enabled` and also do settings migration.